plugins/pastefromword/filter/default.js

```diff
@@ -461,6 +461,17 @@
 					// 		Address of the resource as a valid URL. At least one of data and type must be defined.
 					// If there is not `data`, skip the object element. (https://dev.ckeditor.com/ticket/17001)
 					return !!( element.attributes && element.attributes.data );
+				},
+
+				// Integrate page breaks with `pagebreak` plugin (#2598).
+				'br': function( element ) {
+					var styles = tools.parseCssText( element.attributes.style );
+
+					if ( editor.plugins.pagebreak && styles[ 'page-break-before' ] === 'always' ) {
```

Comment:
`editor.plugins.pagebreak` should be moved to the beginning of the function, as an early return. If the plugin is no loaded, there is no need to even parse element's styles.